user-guide/Troubleshooting/Known_issues/KI_ButtonState_shapes_in_Visual_Overview_fail_to_hide_as_expected.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: KI_ButtonState_shapes_in_Visual_Overview_fail_to_hide_as_expected
+---
+
+# ButtonState shapes in Visual Overview fail to hide as expected
+
+## Affected versions
+
+- DataMiner 10.4.8, if your Cube client does not have internet access.
+
+- Cube clients that have been enforced to use Cube version 10.4.2425.2536 or lower, which can be configured using the *Force a specific version* setting in *System Center* > *System settings* > *Manage client versions*.
+
+## Cause
+
+The method of displaying images in Visual Overview was modified to prevent freezes. However, during this change, an oversight occurred, leading to a rendering fault specifically affecting *ButtonState* shapes.
+
+## Fix
+
+- For Cube clients without internet access:
+
+  1. Request an upgrade package from an Administrator working on a DMS that has not been impacted by this issue.
+
+  1. Manually upgrade your DMA using the provided package. See [Upgrading a DataMiner Agent](xref:Upgrading_a_DataMiner_Agent).
```

Comment:
There currently is no upgrade package currently that has a fix, the revert will be part of the 10.4.9 upgrade package